apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -59,6 +59,11 @@ type ContourSettings struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// Enable debug logging.
+	//
+	// +optional
+	Debug bool `json:"debug,omitempty"`
```

Comment:
personally would prefer using this type https://github.com/projectcontour/contour/blob/491e263e2ab3f4b6e6c0fa629b580491a03ab2da/apis/projectcontour/v1alpha1/contourconfig.go#L297-L306

rather than a boolean here

something like:

```suggestion
	LogLevel LogLevel `json:"debug,omitempty"`
```

Reply:
with a default of `info`

Reply:
i just use `bool` here, is just want to match the 
```
	serve.Flag("debug", "Enable debug logging.").Short('d').BoolVar(&ctx.Config.Debug)
```
in serve.go, if you think `LogLevel` is better, I will change it

Reply:
same as the other PR:

yeah I would prefer the rename: personally don't love the existing flag name and I would like to new field to be a bit more readable

no need to match the field name exactly with the flag, as people using the provisioner don't need to know the exact flag name, we can make it configurable with something more clear what the meaning is

Reply:
done